src/components/Layout.js

```diff
@@ -54,7 +54,7 @@ class Layout extends React.Component {
   render() {
     return (
       <DocumentTitle>
-        <LoadingOverlay loading={!this.props.firstLoadComplete}>
+        <LoadingOverlay loading={this.props.loadingClustersList !== false}>
```

Comment:
Can this be `this.props.loadingClustersList` instead of `this.props.loadingClustersList !== false` ?

Reply:
Changed: the only downside is that now flags have to be initialized to avoid undefined flags.

Reply:
Oh I'm ok if this is done with a reason. It just read a bit weird to me.